src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1041,6 +1043,14 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
         return "";
     }
 
+    @computed get alterationTypesInQuery() {
+        if (this.props.store.selectedMolecularProfiles.isComplete) {
+            return _.uniq(this.props.store.selectedMolecularProfiles.result.map(x=>x.molecularAlterationType));
```

Comment:
@alisman in the last PR you commented https://github.com/cBioPortal/cbioportal-frontend/pull/1400#discussion_r212399824

So you want me to wrap line 1048?